astropy/io/fits/diff.py

```diff
@@ -1472,56 +1463,6 @@ def diff_values(a, b, rtol=0.0, atol=0.0):
         return a != b
 
 
-def report_diff_values(fileobj, a, b, ind=0):
```

Comment:
I am not sure if it is okay to just remove this (not backward compatible), or to raise deprecation warning, or to make it indefinitely backward compatible. This is not exposed to typical users.

Reply:
On github the only usage of this found only in forks/copies of pyfits or astropy, so I'm confident it's unlikely to bite too many people.

Reply:
It was an internal thing, so no need for a deprecation warning imo.